setup.sh

```diff
@@ -15,6 +15,10 @@ sudo apt-get install -y postgresql postgresql-contrib
 echo "install requirements"
 pip3 install -r /vagrant/requirements.txt
 
+echo "running nunjucks to the Fronyend"
+sudo apt-get install nunjucks
```

Comment:
Are you sure you need this, given that Flask includes Jinja2? It better not to mix two template engines in the same project. Maintaining the templates can become very confusing...

Reply:
you are right, I tested and I don't need this :), 
i didn't know that the flask include Jinja2 :)